src/styles/atoms/_atoms.avatar.scss

```diff
@@ -33,7 +47,7 @@
   }
 }
 
-a .a-avatar {
+a.a-avatar {
```

Comment:
Volgens mij is dit een breaking change.

Reply:
Correct, dat was niet de bedoeling. Goed gezien! 😄